src/services/SpecStore.ts

```diff
@@ -15,6 +16,7 @@ export class SpecStore {
   externalDocs?: OpenAPIExternalDocumentation;
   contentItems: ContentItemModel[];
   securitySchemes: SecuritySchemesModel;
+  'x-webhooks'?: WebhookModel;
```

Comment:
What about `xWebhooks` or `xWebHooks`?

Reply:
This is a convention by OpenAPI spec, all vendor extensions should prefixed by `x-`.

Reply:
Oops, my bad. This is our code not typing for OAS. @stasiukanya could you update it to `xWebhooks`? And everywhere else.

Reply:
Why not `webhooks` then if it is our code?

Reply:
@adamaltman good catch!